inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/nonzero.cpp

```diff
@@ -16,10 +16,25 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 
+class MyriadNonZeroLayerTest : public NonZeroLayerTest {
+protected:
+    void SetUp() override {
+        configuration[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
```

Comment:
I think you can modify NonZeroLayerTest to pass `ConfigMap` like it has been done in Eltwise tests.
Thus there be no need to create heir class

Reply:
Good catch! Thx. It fixed.